tests/e2e/commands/subnet.go

```diff
@@ -23,17 +23,66 @@ const (
 )
 
 /* #nosec G204 */
-func CreateSubnetEvmConfig(subnetName string, genesisPath string) (string, string) {
+func CreateSubnetEvmConfigNonSOV(subnetName string, genesisPath string) (string, string) {
```

Comment:
may be better to split commands along sov and nonsov subdirs? and avoid all the sufices ?